ui/plugins/bookmarks/src/components/entry-renderer.tsx

```diff
@@ -62,14 +68,53 @@ const EntryCardRenderer = (props: IEntryCardRendererProps) => {
   const postReq = usePost({ postId: itemId, enabler: type === ItemTypes.ENTRY });
   const commentReq = useComment(itemId, type === ItemTypes.COMMENT);
 
+  const handleFlipCard = () => {
+    setShowAnyway(true);
+  };
+
   const itemData = React.useMemo(() => {
-    if (type === ItemTypes.COMMENT && commentReq.status === 'success') {
+    if (type === ItemTypes.COMMENT && commentReq.isSuccess) {
       return mapEntry(commentReq.data);
-    } else if (type === ItemTypes.ENTRY && postReq.status === 'success') {
+    } else if (type === ItemTypes.ENTRY && postReq.isSuccess) {
       return mapEntry(postReq.data);
     }
   }, [type, postReq.data, postReq.status, commentReq.data, commentReq.status]);
```

Comment:
you should also update the list of inputs since `postReq.status` is not used inside the function

Reply:
same for `commentReq`